backend/ops_api/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.9-slim
```

Comment:
Do we want to rev to 3.10 here since setup-python is using it?

Reply:
good call 👍